board/linux-additional-modules.config

```diff
@@ -0,0 +1,47 @@
+# Additional modules we want included
```

Comment:
I don't really like the idea of a single config override. Sometimes we want to disable/enable different things for different boards. I do like however to use the buildroot kernel config as a base. Suggestion: keep our override mechanism but use the buildroot version as a base.